src/app/pyproject.toml

```
@@ -72,7 +72,7 @@ version = { attr = "warren.__version__" }
 
 # Third party packages configuration
 [tool.pytest.ini_options]
-addopts = "-v --cov-report term-missing --cov=apps"
+addopts = "-v --cov-config=.coveragerc --cov-report term-missing --cov=apps"
```

The following does not work?

```suggestion
addopts = "-v --cov-report term-missing --cov=apps --omit */tests/*"
```

---

Nope. `pytest` does not recognize this argument